src/breadcrumbs/_base.scss

```diff
@@ -16,26 +15,52 @@
       color: color(blue);
     }
   }
-
-  [data-icon="casper-logo-mark"] {
-    fill: color(text);
-  }
 }
 
 .breadcrumbs-list-item {
   display: inline-block;
-  margin-right: ms(-7);
+  margin-left: ms(-7);
 
-  &:after {
+  &:before {
     content: '>';
-    margin-left: ms(-7);
+    margin-right: ms(-7);
   }
 
-  &:last-child {
-    margin-right: 0;
+  &:first-child {
+    margin-left: 0;
 
-    &:after {
+    &:before {
       display: none;
     }
   }
 }
+
+.breadcrumbs--footer {
```

Comment:
Would be nice to have some documentation above each of the different modifiers so that a new person coming to this file understands what they are used for and when